src/Sylius/Bundle/UserBundle/Tests/DependencyInjection/SyliusUserExtensionTest.php

```diff
@@ -133,6 +133,24 @@ public function it_creates_a_update_user_encoder_listener_for_each_user_type():
         Assert::assertSame('_password', $shopUserListenerDefinition->getArgument(4));
     }
 
+    /** @test */
+    public function it_creates_resetting_token_parameters_for_each_user_type(): void
```

Comment:
```suggestion
    public function it_creates_default_resetting_token_parameters_for_each_user_type(): void
```

Can we also write one test for setting different value?

Reply:
done :slightly_smiling_face: